core/modules/user/user.module

```diff
@@ -2727,17 +2809,35 @@ function _user_mail_notify($op, $account, $language = NULL) {
  * Form element process handler for client-side password hide/show and strength.
  * @param $element
  */
-function user_form_process_password($element) {
+function user_form_process_password($element, $form_state) {
```

Comment:
Isn't this considered an API change (adding a new, required parameter to an existing function)? Can you make it optional instead (like providing a default value)? Also, you'll need to add this new parameter to the docblock above (and make sure it's formatted correctly: https://api.backdropcms.org/doc-standards#param (not like the existing parameter documentation)).

Reply:
> Can you make it optional instead

I'll give it a try.

I need it to get the `$form_state['user']->name` (and mail). Necessary to evaluate the password if an admin (not the current user) edits the form.
This was done wrong before.